app/seeders/review_seeder.rb

```diff
@@ -0,0 +1,25 @@
+class ReviewSeeder
+  REVIEWS = [
+    {
+        dinosaur: Dinosaur.first,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.